src/jumpstart/JumpstartSendConfirmation.tsx

```diff
@@ -25,17 +28,33 @@ type Props = NativeStackScreenProps<StackParamList, Screens.JumpstartSendConfirm
 const TAG = 'JumpstartSendConfirmation'
 
 function JumpstartSendConfirmation({ route }: Props) {
-  const { tokenId, sendAmount } = route.params
+  const { tokenId, sendAmount, serializablePreparedTransactions } = route.params
   const parsedAmount = new BigNumber(sendAmount)
   const { t } = useTranslation()
+  const dispatch = useDispatch()
 
   const token = useTokenInfo(tokenId)
   const usdToLocalRate = useSelector(usdToLocalCurrencyRateSelector)
   const localCurrencyCode = useSelector(getLocalCurrencyCode)
+  const jumpstartSendStatus = useSelector(jumpstartSendStatusSelector)
+
+  useEffect(() => {
+    if (jumpstartSendStatus === 'success') {
```

Comment:
yes you're right, i didn't feel great about this implementation either. this is pretty analgous to clearing jest mocks in the `afterEach` vs `beforeEach` blocks, i do favour ensuring the state is clean at the start rather than the end of a flow (since we may never get to the end, as in your example). i'll tweak this and move the logic to the beginning of the flow

Reply:
the changes i ended up making (also documented in the PR description) -
1. ensure the `depositStatus` is reset to `idle` when a new jumpstart link is created at the beginning of the flow. each link uniquely identifies each transaction.
2. block the "proceed" button on the enter amount screen if we can see there is an inflight or successful transaction already. this means the user needs to go back and launch the flow again, which is all good.